workspace/TS100/src/Translation.cpp

```diff
@@ -770,13 +772,13 @@ const char* SettingsDescriptions[] = {
   /* VoltageCalibration        */ "Kalibracija ulaznog napona. Podešavanje gumbima, dugački pritisak za kraj.",
   /* AdvancedSoldering         */ "Prikazivanje detaljnih informacija tijekom lemljenja.",
   /* ScrollingSpeed            */ "Brzina kretanja dugačkih poruka. <B=brzo, S=sporo>",
-  /* TipModel                  */ "Tip Model selection",
-  /* SimpleCalibrationMode     */ "Simple Calibration using Hot water",
-  /* AdvancedCalibrationMode   */ "Advanced calibration using thermocouple on the tip",
-  /* PowerInput                */ "Power Wattage of the power adapter used",
+  /* TipModel                  */ "Odabir modela lemnog vrha",
+  /* SimpleCalibrationMode     */ "Kalibracija kipućom vodom",
+  /* AdvancedCalibrationMode   */ "Kalibracija korištenjem termo-elementa",
+  /* PowerInput                */ "Snaga modula za napajanje",
```

Comment:
This got updated automatically, without me touching the translation strings. I believe it's because `Translation.cpp` wasn't regenerated in #405.